### PR TITLE
Added user's classCode

### PR DESCRIPTION
**Pull request by 3normousz**
- It should be the same as Compulsory & Elective class code

### CODE REVIEW

**Comment by ImJustChew** [comment]
Move it to its own function for now, as a server action, such that we don't pollute the JWT with extra information.

**Comment by ImJustChew** [comment]
Could you seperate this into its own function, like place it in `lib/headless_ais/courses.ts` as a `getClassDetailed(ACIXSTORE: string)` , we will then use it when we really need the data.

**Comment by 3normousz** [comment]
Ok sure

File: src/lib/headless_ais/courses.ts [hunk 1]
```diff
@@ -228,4 +228,68 @@ export const getHiddenCourseSelectionCode = async (ACIXSTORE: string, dept: stri
...
+          "sec-ch-ua-mobile": "?0",
+          "sec-ch-ua-platform": "\"Windows\"",
+          "sec-fetch-dest": "document",
+          "sec-fetch-mode": "navigate",
+          "sec-fetch-site": "same-origin",
+          "sec-fetch-user": "?1",
+          "upgrade-insecure-requests": "1",
+          "referer": `https://www.ccxp.nthu.edu.tw/ccxp/INQUIRE/IN_INQ_STU.php?ACIXSTORE=${ACIXSTORE}`
+        },
+        keepalive: true,
```
Comment (ImJustChew) [inline src/lib/headless_ais/courses.ts:250]: Remove keep alive to prevent mem leak

**Review by ImJustChew — APPROVED** [review]

**Pull request merged**